Libraries/LibWeb/Encoding/TextEncoderStream.idl

```diff
@@ -0,0 +1,11 @@
+#import <Encoding/TextEncoder.idl>
+#import <Streams/GenericTransformStream.idl>
+
+// https://encoding.spec.whatwg.org/#textencoderstream
+[Exposed=*]
+interface TextEncoderStream {
+    constructor();
+};
+
+TextEncoderStream includes TextEncoderCommon;
```

Comment:
:thinking:  TextEncoderCommon should really be in its own file
(I had that laying around in a branch for a while if you wanted to yoink that here and make use of it https://github.com/trflynn89/ladybird/commit/342cb8892f493454b67e069d916396307226af49)